src/components/job/JobView.jsx

```diff
@@ -78,6 +79,12 @@ export default function JobView() {
       <div className="h-full w-full flex flex-col justify-between">
         <div className="flex flex-col card">
           <div className="card-body items-center">
+            {data.adminMessage && auth.user.role === ROLES.EMPLOYER ? (
+              <span>
+                <p className="font-bold">Administrator Feedback:</p>
+                <p>{data.adminMessage}</p>
+              </span>
+            ) : null}
```

Comment:
guess should be one more condition: job is rejected

Reply:
add style for it if you have time?

Reply:
I added a status badge for style and only showing admin feedback if it's there